.eslintrc.js

```diff
@@ -15,16 +15,53 @@
  */
 module.exports = {
   'root': true,
+  'env': {
+    'node': true,
+  },
+  'plugins': ['header'],
+  'rules': {
+    'header/header': [
+      2,
+      'block',
+      [
+        '*',
+        {
+          pattern:
+            ' * (© Copyright IBM Corporation 20\\d\\d(?:, 20\\d\\d)?\\. All Rights Reserved\\.|\\(C\\) Copyright IBM Corp\\. 20\\d\\d\\.)',
+          template: ' * © Copyright IBM Corporation 2021. All Rights Reserved.',
+        },
+        ' *',
+        ' * Licensed under the Apache License, Version 2.0 (the "License");',
+        ' * you may not use this file except in compliance with the License.',
+        ' * You may obtain a copy of the License at',
+        ' *',
+        ' *      http://www.apache.org/licenses/LICENSE-2.0',
+        ' *',
+        ' * Unless required by applicable law or agreed to in writing, software',
+        ' * distributed under the License is distributed on an "AS IS" BASIS,',
+        ' * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.',
+        ' * See the License for the specific language governing permissions and',
+        ' * limitations under the License.',
+        ' ',
+      ],
+    ],
+    'max-len': ['error', { 'code': 100 }],
+  },
   'overrides': [
     {
-      'env': {
-        'node': true,
-      },
-      'files': ['**/*.js', '**/*.jsx'],
+      'files': ['**/*.js'],
       'plugins': ['node', 'jest', 'prettier'],
-      'extends': ['airbnb-base', 'plugin:jest/recommended', 'plugin:jest/style', 'prettier'],
+      'extends': [
+        'airbnb-base',
+        'plugin:import/recommended',
+        'plugin:jest/recommended',
+        'plugin:jest/style',
+        'prettier',
+      ],
       'rules': {
         'camelcase': 'off',
+        'no-console': 'off',
```

Comment:
Setting this `off` means, the code we write can remain _unintentional_ console logs.
To avoid this we should not set this globally, only where it is necessary (like README integration tests).

Reply:
Changed in 4ce616e